src/cluster_topology_observer.py

```diff
@@ -129,14 +130,16 @@ def main():
 
     Watch the Patroni API cluster info. When changes are detected, dispatch the change event.
     """
-    patroni_url, verify, run_cmd, unit, charm_dir = sys.argv[1:]
+    patroni_url, verify, run_cmd, unit, charm_dir, patroni_password = sys.argv[1:]
```

Comment:
Passing the pass as an argument, which is not very safe. In principle, the old behaviour should keep on working as long as patroni doesn't protect the get calls. Should we keep the auth here?